server/src/test/java/org/elasticsearch/search/aggregations/AggregationsTests.java

```diff
@@ -138,7 +137,6 @@ public class AggregationsTests extends ESTestCase {
         new InternalDateRangeTests(),
```

Comment:
I think we want this test also in the new module? As we move over aggregations we remove entries here and add it to the equivalent test case in  aggregations module?

Reply:
How do we think this should look in the end? Would there be an `AggregationsTests` in `test:framework` that has no aggs listed? That feels maybe not great. But neither does having `test:framework` depend on the module. Maybe it'd make sense to move this class to a `aggregations:test` module that other aggregation extensions depend on. That'd pick up all these tests.

Reply:
But in the short term removing this from the list is fine.

Reply:
> Maybe it'd make sense to move this class to a aggregations:test module that other aggregation extensions depend on. That'd pick up all these tests.

Yes, I think when enough tests (50%?) have moved we should move this test to the aggregation module.

Reply:
:+1: